### DISCUSSION TITLE
VoteFeature - Public catalog with feature voting

### DISCUSSION THREAD

**Discussion by vladpalos** [pocketbase/pocketbase#5571] [Show and tell]
Hello everyone, 

I just want to tell you about a web app I've worked on for a bit, that is backed by PocketBase and Next.js. 

It's called **VoteFeature** and it's a simple public catalog and product management tool  with feature voting capabilities. 

[https://votefeature.com](https://votefeature.com)

PocketBase was a breeze to set up and use. Massive thanks for this great tool.

**Comment by benallfree** [comment]
Testing notes:

* Signup redirects to http://localhost:3000/my-products/new
* Upload custom logo, but it doesn't show
* No way to edit logo later
* Suggesting a feature does not work. The 'suggest feature' button doesn't respond

After you get this stuff working, I'll send it out to the 10k pockethost.io users :)

Suggestions:
* Make max votes per user. So if I set the project to max 3 votes, each user can allocate all 3 votes to a single featur or spread them around on up to 3 feature suggestions. This encourages users to prioritize rather than simply upvoting everything that sounds nice. The votes can be moved but they once a suggestion is accepted the votes are locked until the suggestion is canceled or closed by admin. Then those votes are released again.

**Reply by vladpalos** [reply]
@benallfree Amazing, thank you very much for your notes and suggestions! I'll get right on fixing it.

**Reply by vladpalos** [reply]
Hi @benallfree, the issues should be now fixed. Please let me know if you're having any other problems.
Regarding the max votes, I think it's a great suggestion. I'll need a bit of time to think how to implement it so it's intuitive enough.

**Comment by benallfree** [comment]
Submit feature is still broken. Nothing saves.

Please consider testing each deployment end to end as a fresh user until things are stable. It can be discouraging for new users to try the service and even the most basic thing does not work :)

**Reply by vladpalos** [reply]
Thanks @benallfree. You're right, there are no tests at the moment. I'm just about to add some playwright tests for most functionality. I'll let you know when they're fixed.

**Reply by vladpalos** [reply]
Hi Ben @benallfree  I'm having a trouble reproducing this specific issue. Would you mind elaborating a bit? 

Is this happening with your own user when you go to the Product Page (pockethost) and press the Suggest Feature button? 

This should send a suggesting in the "Suggestions" Manage tab. From there, you can convert that into a feature on your backlog.

**Reply by benallfree** [reply]
Let's move off the PB forum, do you have a github repo for this? (Your Discord invite doesn't work)

**Reply by vladpalos** [reply]
I fixed the Discord link. You can also send an email to hello@votefeature.com. 
I also created an empty repository if that's better: https://github.com/vladpalos/votefeature

**Comment by FoolsTech** [comment]
Looks good man :) Nice design skills. 
But only three products :D 
You gave it up or still working on more cool features?

**Reply by OpenGreenStreet** [reply]
Sorry: How can this solution be used in other Github forums
